client/src/components/Form/BlogsInputs/BlogsMarkdownInput.jsx

```diff
@@ -0,0 +1,21 @@
+import React from 'react';
+import FormField from '../../common/FormField';
+import TextAreaAutoResize from '../../common/TextAreaAutoResize';
+import classes from './BlogsInputs.module.css';
+
+const BlogsMarkdownInput = (props) => {
+    return (
+        <FormField
+        control={ TextAreaAutoResize }
+        label="Enter Blog Content"
+        placeholder="Tell us more about this blog..."
```

Comment:
very poor placeholder ![bitmoji](https://render.bitstrips.com/v2/cpanel/5a7f0ed1-7305-4fcf-9bda-18469557727a-f45fca63-6c12-4e74-9dda-cd7dc0018281-v1.png?transparent=1&palette=1&width=246)

Reply:
We can change it later